### ISSUE TITLE
TVT reported inconsistent

### ISSUE THREAD

**Issue by sunchipsster1** [deepmind/deepmind-research#308]
Hi guys! Was trying out the Temporal value transport code you guys kindly provided at https://github.com/deepmind/deepmind-research/tree/master/tvt

While I got matching results to the Key-to-door --> with last phase reward reaching 10 very quickly when TVT is enabled, **_I was unable to get a moving average of more than 4 or 5 for  last phase reward in the Active-visual-match_** even after many many iterations (with TVT enabled).

Are the provided **_hyperparameters_** provided by default in the repository as given -- correct for the Active-visual-match task? Or do they need to be modified? 

Thank you in advance for your kind help! :)

**Comment by joshabramson** [comment]
Hi! Thanks for your interest in our work :)

I tried a run with 6 repeats and 5 out of 6 achieved a score of 10 by 2M steps. It is possible that some runs occasionally fail to train, could you try running another one? The hyperparameters are meant to work for both levels.

**Comment by sunchipsster1** [comment]
@joshabramson thank you so much for the kind help! 
I wonder if I may have misinterpreted what is meant by 2M steps? -- the way I had interpreted 2M steps was number of steps = (ep_length = 120) * (batch_size = 16) * ep_num . So in effect, 2M steps **requires ep_num ~ 1000 or so.**  

Or does the 2M steps really mean that you trained it for **ep_num > 2M?** 

Thanks in advance for the kind clarification! :)

**Comment by joshabramson** [comment]
Apologies for calling it 'steps', that just corresponds to the tensorboard terminology. i had forgotten that we set it up so that the x-axis is the total number of episodes. where we do the logging to tensorboard it says:

        # Tensorboard x-axis is total number of episodes run.
        summary_writer.add_summary(summary, ep_num * batch_size)

So it is 2M episodes in total. It takes about 2 seconds to run an episode at batch size 16 for a P4 Nvidia gpu (about 1k environment steps per second), which means we get to 2M episodes in just under 3 days.

**Comment by sunchipsster1** [comment]
Thanks so much, this is totally clear! Happy holidays :)
